package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -0,0 +1,306 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
So for a documentation page to get built you also need to add a rst stub in `package/doc/sphinx/source/documentation_pages/analysis` (look at others in that dir) and a link to the page in `documentation_pages/analysis_modules.rst`

Each rst file becomes a page at docs.mdanalysis.org, and it's the `automodule` thing in the rst that basically just points to this source here for content.